### ISSUE TITLE
Import timesheets using console fails to process tags

### ISSUE THREAD

**Issue by synaestic** [kevinpapst/kimai2#1877]
I've tried to import timesheets but the import command doesn't handle the entries in the "Tags" field properly and throws an exception.

How to reproduce:

1. Add two timesheets with tags.
2. Export as CSV.
3. Delete the last line `"","","","[00]:00","0","0","","","","","","","","",""`
4. Import the file

```
Kimai importer: Timesheets
==========================

                                                                                                                        
 [ERROR] Failed importing timesheet row 2 with: An exception occurred while executing 'INSERT INTO kimai2_tags (name,   
         color) VALUES (?, ?)' with params ["my tag", null]:                                                            
                                                                                                                        
         SQLSTATE[23000]: Integrity constraint violation: 1062 Duplicate entry 'my tag' for key 'UNIQ_27CAF54C5E237E06'
```
For me it seems the tag progression at [src/Command/ImportTimesheetCommand.php#L311](https://github.com/kevinpapst/kimai2/blob/master/src/Command/ImportTimesheetCommand.php#L311) lacks a `TagRepository::findIdsByTagNameList` implementation.

**Comment by kevinpapst** [comment]
Thanks @synaestic for posting the issue.

Seems you already identified the issue and fix: are you up for a fix / PR?

**Comment by synaestic** [comment]
I hope to have some spare time soon.

**Comment by kevinpapst** [comment]
Fixed in #1883 by @synaestic 👍 closing!
